test/errors/security_error_test.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: constant_identifier_names
```

Comment:
I think the constants can be renamed to follow the standard since it's just a test

Reply:
Here as well, renaming/deprecating is something i would leave up to you if that's ok.